blog/let's-roll.mdx

```diff
@@ -0,0 +1,59 @@
+---
+slug: Let's Roll- Rollkit's community call
+title: "Let's Roll- Rollkit's community call"
+authors: [AlCJoseph]
+tags: [Rollkit]
```

Comment:
```suggestion
tags: [Rollkit][Community]
```

Reply:
tag here is not well written, it should be [Rollkit, Community]